public/js/index.js

```diff
@@ -0,0 +1,83 @@
+const cells = document.getElementsByClassName('cell');
+const scores = document.getElementsByClassName('player-score');
+const namesDisplay = document.querySelectorAll('.name');
+const message = document.querySelector('.msg');
+
+const print = (element) => {
+  const domElement = element;
+  element.addEventListener('click', () => {
+    // eslint-disable-next-line no-undef
```

Comment:
[Optional]

Why not use the same file for the two javascript files `index.js` and `logic.js`?
Since ES6 `import/export` is out-of-scope for this project. I think it's okay to use a single file for the whole javascript as the main objective is to use **the Factory Functions and the Module Pattern**.

This will help with the `eslint no-undef` and `eslint no-unused-vars` rules.

In future projects, using webpack and ES6 `import/export`,  the code will be divided strategically between different directories and files.